src/tw-install.rc

```diff
@@ -0,0 +1,17 @@
+### tw-install.rc configuration file 
+TASKDIR=~/.task
+TASKDDIR=/var/task
+PLATFORM=linux
+MOBILE_PLATFORM=android
+MOBILE_SOURCE=fdroid
+PACKAGE_MANAGER=apt
+HOOK_DIR=~/.task/hooks
+TMP_DIR=~/tmp
+TASKWIKI_DIR=~/.task/wiki
+TASKNOTES_DIR=~/.task/notes
+TASKTHEME_DIR=~/.task/theme
+BIN_DIR=~/.local/bin
```

Comment:
Would you perhaps be able to explain what the bin dir is for?
*Possibly as a comment, such that others coming into this repository also know what it is for?

Reply:
This is just a sample code, you can let me know what to put in place of these

Reply:
Thank you for your response, I have looked up the source of the sample code, and assume this is that source: https://github.com/HiveMinds/tw-install/issues/83 and asked @linuxcaffe if he would be so kind to provide an elaboration on the two commented entries. 

To answer your suggestion, I think the current list is a nice start (assuming a meaningful interpretation is given for the two commented entries in question:
```
BIN_DIR=~/.local/bin
```
and 
```
WEB_DIR=~/var/www/something
```
And I would like to extend that list with:

```
LOG_LOCATION=src/logs/
TASKSERVER_TAR_NAME=taskd-1.1.0
TW_USERNAME=First
TW_ORGANISATION=Public
PORT=53589
TASKDDATA=/var/taskd
````
It would be nice if this list, which comes from the `hardcoded_variables.txt` could be removed from there, and if the respective loading of those variables is changed from `hardcoded_variables.txt` to `tw-install.rc`.